packages/astro/test/units/dev/restart.test.js

```diff
@@ -133,7 +133,7 @@ describe('dev container restarts', () => {
 			cmd: 'dev',
 			logging: defaultLogging,
 		});
-		const settings = createSettings(astroConfig);
+		const settings = createSettings(astroConfig, 'dev');
```

Comment:
isn't the second arg the cwd?

Reply:
Not anymore: https://github.com/withastro/astro/blob/b46dfdb12fae70fb6f126b6105968d371073ea63/packages/astro/src/core/config/settings.ts#L113